src/entities/SaveHistory.ts

```diff
@@ -0,0 +1,83 @@
+import * as env from "@utils/config.constant";
+import fs from 'fs';
+
+interface HistoryEntry {
+  Epoch: number;
+  
+}
+
+export class SaveHistory {
```

Comment:
Also, it seems like this class can have just static methods? Like `History.createEntry(...)` instead of `const history = new History(); history.createEntry(...)`. Not sure we actually need an instantiated class for History.